### ISSUE TITLE
licence

### ISSUE THREAD

**Issue by RMHogervorst** [cboettig/lifxr#4]
I think you can't actually use the cc0 licence , probably a MIT licence would be better.

**Comment by cboettig** [comment]
I'd be happy to switch, but I think you'll find CRAN recognizes the use of CC0,

```r
available.packages(fields = "LICENSE") %>% 
as_data_frame() %>% 
select(Package, License, Repository) %>% 
filter(License == "CC0")
```

**Comment by RMHogervorst** [comment]
You are absolutely right of course, what I meant, and I should write a bit less terse, is that I thought the creative Commons licenses are not meant for code. Choosalicence.com gives cc as a option for if you don't have software. 

This is a minor point though

**Comment by cboettig** [comment]
https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software

CC0 is GPL compatible according to FSF.  Technically CC0 isn't a license but rather a public domain definition, and one which has paid more attention to international legal language than the very terse MIT. ( Given that R packages can distribute data, vignettes etc as well as source code, it's not always obvious what the source code license of the package covers, though that's no issue here).

On the other hand, OSI rejected the current CC0 license after lengthy debate due to patent questions, so perhaps we should switch to something more conventional. Being at Berkeley I'm partial to BSD-2 ;-)

**Comment by RMHogervorst** [comment]
All right =-)
